build_tools/ci/run_matmul_test.sh

```diff
@@ -653,14 +653,15 @@ run_matmul_test_on_shapes ${i32_shapes_small[@]} \
     --acc_type "i32" \
     --num_repeat_runs "10"
 
-run_matmul_test_on_shapes ${i32_shapes_small[@]} \
-    --name_prefix "small" \
-    --lower_to_aie_pipeline "objectFifo" \
-    --tile_pipeline "pack-peel" \
-    --lhs_rhs_type "i32" \
-    --acc_type "i32" \
-    --num_corruption_repeat_runs "10" \
-    --enable_packet_flow "true"
+# Disable because of failure on 4x4 AIE array
```

Comment:
Compilation or runtime? 

If compilation, please provide more info in the comment  
If runtime, use --num_repeat_runs 0

Reply:
It's the compilation error, the error information is in the description. Jorn will be working on it next week. I'd prefer not include the detail error message in the comments here.